src/main/java/seedu/realtime/model/util/SampleDataUtil.java

```diff
@@ -95,8 +96,8 @@ public static Listing[] getSamepleListings() {
 
 
 
-    public static ReadOnlyAddressBook getSampleAddressBook() {
-        AddressBook sampleAb = new AddressBook();
+    public static ReadOnlyRealTime getSampleRealTime() {
+        RealTime sampleAb = new RealTime();
```

Comment:
```suggestion
    public static ReadOnlyRealTime getSampleRealTime() {
        RealTime sampleRt = new RealTime();
```

Doing this means that the next few lines of code has to change from `sampleAb` to `sampleRt`